db/python/tables/family.py

```diff
@@ -288,19 +397,20 @@ async def get_id_map_by_external_ids(
     async def get_id_map_by_internal_ids(
         self, family_ids: List[int], allow_missing=False
     ):
-        """Get map of {external_id: internal_id} for a family"""
+        """Get map of {internal_id: external_id} for a family"""
         if len(family_ids) == 0:
             return {}
-        _query = 'SELECT id, external_id FROM family WHERE id in :ids'
+        # FIXME needs to be uniqueified!!
```

Comment:
Does this FIXME still need to be FIXME'd?

Reply:
Yes indeed!

Reply:
I checked the two places where this function is invoked, which turn out to mostly just want a single external ID for use in a pedigree or similar. Hence the best approach, at least in the short term, is indeed to uniqueify this by returning only the **primary** external ids — which is now implemented.